package.json

```diff
@@ -8,7 +8,7 @@
     "preferGlobal": true,
     "private": true,
     "engines": {
-        "node": ">= 12.10.0"
+        "node": ">= 14.14.0"
```

Comment:
Happy to raise this, but we'll need to do a major version update, i'll add that in

Reply:
Makes sense. Do you think you'd do a major version update with the three PRs I submitted today? I was planning to use Droppy for a hobby project, which is why I was testing and fixing things. Just trying to gauge if I should do a temporary workaround in my other project or not. Thanks!

Reply:
We can sort it out quickly I'm just a bit busy with work lately :) I'll look more in depth soon